content/chapters/compute/lecture/slides/intro.md

```diff
@@ -0,0 +1,74 @@
+# COMPUTE
+
+1. [Processes](#1-processes)
+1. [Threads](#2-threads---backstory)
+1. [Scheduling](#3-scheduling)
+1. [Synchronization](#4-synchronization)
+
+---
+
+### Backstory
+
+Apps running while you are doing your homework:
```

Comment:
`Apps` doesn't include `The OS itself`. We could leave it like this or change `Apps` to `Software`.